webapp/src/components/expanded_view/component.tsx

```diff
@@ -137,8 +137,6 @@ const StyledMediaFullscreenButton = styled(MediaFullscreenButton)`
 	background-color: transparent;
 `;
 
-const MaxParticipantsPerRow = 4;
```

Comment:
Where did this go?

Reply:
It wasn't being used, so I removed it.

Reply:
That must be a mistake or a previous merge issue because I remember adding it to both components. I guess it doesn't hurt to bring it back here:

https://github.com/mattermost/mattermost-plugin-calls/blob/f8d4c0918f92fe72a34bad5b501e165c80f730f5/webapp/src/components/expanded_view/component.tsx#L816